.ci/bump-release-version.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+checkForVariable() {
+  VAR_NAME=$1
+  if [ ! -v "$VAR_NAME" ]; then
+    echo "Error: Define $1 environment variable"
+    exit 1
+  fi
+}
+
+checkForVariable "READ_ONLY_TOKEN"
+
+CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
+git fetch --prune --unshallow
+NEW_RELEASE=$(git describe --always --abbrev=0 | cut -d '-' -f 2)
+
+echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
+echo NEW_RELEASE="$NEW_RELEASE"
+
+SET_VERSION="$NEW_RELEASE-SNAPSHOT"
+echo SET_VERSION="$SET_VERSION"
+
+echo "bump version in pom.xml"
+mvn -e --no-transfer-progress versions:set -DgroupId=com.puppycrawl.tools -DartifactId=checkstyle \
+ -DoldVersion="${CS_RELEASE_VERSION}" -DnewVersion="${SET_VERSION}"
+git push origin master
+
+echo "Updating milestone at github"
+MILESTONE_ID=$(curl -s \
+                -X GET https://api.github.com/repos/checkstyle/checkstyle/milestones?state=open \
+                | jq ".[0] | .number")
+curl \
+  -X PATCH \
+  -H "Authorization: token $READ_ONLY_TOKEN" \
+  https://api.github.com/repos/checkstyle/checkstyle/milestones/"$MILESTONE_ID" \
+ -d "{ \"title\": \"$NEW_RELEASE\", \
+        \"state\": \"open\", \
+        \"description\": \"\" \
```

Comment:
can we keep description without changes ?

Reply:
Changes Done